### PR TITLE
feature(onboarding): adds new modal for onboarding

### PR DESCRIPTION
**Pull request by ccabrera35**
## Description
- Adds a new cookie `isOnboarded` (defaults to false) when a user logs in.
  - When a user goes to the dashboard page it will check if the cookie exists, if the value is falsy the user will be shown a modal with instructions about modifying their alerts

### CODE REVIEW

**Comment by ccabrera35** [comment]
Should I not have committed the lock file?

File: src/components/alerts/onboard.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        <Button
+          onClick={() => {
+            setModal({ open: false })
+            router.push('/alerts')
+            completeOnboarding()
+          }}
+          variant='contained'
+          className={classes.normal}
+        >
+          Take me there
```
Comment (ccabrera35) [inline src/components/alerts/onboard.tsx:121]: Hardcoded this here let me know what you think?

Comment (j-mendez) [reply src/components/alerts/onboard.tsx:121]: This works

**Review by j-mendez — APPROVED** [review]
Good PR, for making this modal appear and having it navigate to another view I think it may make sense to trigger after the first website add on the dashboard. If we want this to pop up initially it would be nice to have the component on the alerts page on the modal. Maybe moving the alerts page method to a component and import it in both sections?

**Comment by j-mendez** [comment]
> Should I not have committed the lock file?

we need the lockfile in the repo. One of the packages most likely `^` updated from the version

**Pull request merged**